interface/themes/style_manila.css

```diff
@@ -265,9 +265,13 @@ a:hover {
 .titlebar { height: 20px; }                 /* $GLOBALS['titleBarHeight'] */
 
 
+.navbar-custom {
+    padding:0px;
+}
+
 /* style taken out of the addressbook code and put here instead */
 #addressbook_list td {
-font-family: 'FontAwesome', Arial, Helvetica, sans-serif;
+font-family: 'source_sans_pro','FontAwesome', Arial, Helvetica, sans-serif;
```

Comment:
TODO
I think I messed up on the merge and removed this source_sans_pro from a line. Will add it back.